exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,370 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "It is possible to implement this exercise without concurrency,"
+  ],
+  "cases": [
+    {
+      "uuid": "983a1528-4ceb-45e5-8257-8ce01aceb5ed",
+      "description": "Newly opened account has zero balance",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "balance"
+          }
+        ]
+      },
+      "expected": 0
+    },
+    {
+      "uuid": "e88d4ec3-c6bf-4752-8e59-5046c44e3ba7",
+      "description": "Single deposit",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "deposit",
+            "value": 100
+          },
+          {
+            "operation": "balance"
+          }
+        ]
+      },
+      "expected": 100
+    },
+    {
+      "uuid": "3d9147d4-63f4-4844-8d2b-1fee2e9a2a0d",
+      "description": "Multiple deposits",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "deposit",
+            "value": 100
```

Comment:
Maybe we can now use `amount` here as we've dropped it as the operation? This would also apply to the `deposit` actions.

Reply:
Did I do as you intended in the commit?

Reply:
You did!